changes/test_tasks.py

```diff
@@ -311,6 +312,30 @@ def test_no_message_found(self, mock_create_outbound):
         )
         self.assertFalse(mock_create_outbound.called)
 
+    @mock.patch("changes.tasks.utils.ms_client.create_outbound")
+    @responses.activate
+    def test_timeout_message_sent_delivered(self, mock_create_outbound):
+        """
+        The task should send the correct outbound based on the delivered event.
+        """
+        self.create_outbound_lookup()
+        self.create_identity_lookup()
+
+        process_whatsapp_timeout_system_event("messageid", 1542844800, "undelivered")
+
+        mock_create_outbound.assert_called_once_with(
+            {
+                "to_identity": "test-identity-uuid",
+                "content": (
+                    "We see that your MomConnect WhatsApp messages are not being "
+                    "delivered. If you would like to receive your messages over "
+                    "SMS, reply ‘SMS’."
+                ),
+                "channel": "JUNE_TEXT",
+                "metadata": {},
+            }
+        )
```

Comment:
We probably want at least two tests here, one that sends an SMS message, and one that doesn't

Reply:
will do